store/src/server/api.rs

```diff
@@ -127,7 +127,11 @@ impl api_server::Api for StoreApi {
 
         let notes = request.notes;
 
-        let _ = self.state.apply_block(block, nullifiers, accounts, notes).await;
+        let apply_block_status = self.state.apply_block(block, nullifiers, accounts, notes).await;
+
+        if let Err(err) = apply_block_status {
+            println!("Error: {:?}", err);
+        }
```

Comment:
Was this a temp change? If so, we should revert it and return the error. If we want to add logging, we should probably use the tracer.